spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=11,>=500000,4,11
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=12,>=500000,4,12
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=13,>=100000,4,13
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=14,>=50000,4,14
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=15,>=2000,4,15
+"cemetery;commercial;golf_course;hospital;industrial;plant;generator;substation;railway;sports_centre;recreation_ground;retail;stadium;zoo;wildlife_park;winter_sports;pier;wastewater_plant,works;bridge;tower;breakwater;water_works;groyne;dike;cutline",,>=16,+,4,16
+theme_park;resort;aquarium;winery;maze;beach,,>=10,>=1000000,5,10
+theme_park;resort;aquarium;winery;maze;beach,,>=11,>=400000,5,11
+theme_park;resort;aquarium;winery;maze;beach,,>=12,>=200000,5,12
+theme_park;resort;aquarium;winery;maze;beach,,>=13,>=50000,5,13
+theme_park;resort;aquarium;winery;maze;beach,,>=14,>=20000,5,14
+theme_park;resort;aquarium;winery;maze;beach,,>=15,>=2000,5,15
+theme_park;resort;aquarium;winery;maze;beach,,>=16,+,5,16
+garden;allotments;pedestrian;common;pitch;place_of_worship;playground;school;attraction;artwork;wilderness_hut;hanami;tree_row;hedge,,>=12,>=500000,6,12
+garden;allotments;pedestrian;common;pitch;place_of_worship;playground;school;attraction;artwork;wilderness_hut;hanami;tree_row;hedge,,>=13,>=100000,6,13
+garden;allotments;pedestrian;common;pitch;place_of_worship;playground;school;attraction;artwork;wilderness_hut;hanami;tree_row;hedge,,>=14,>=50000,6,14
+garden;allotments;pedestrian;common;pitch;place_of_worship;playground;school;attraction;artwork;wilderness_hut;hanami;tree_row;hedge,,>=15,>=2000,6,15
+garden;allotments;pedestrian;common;pitch;place_of_worship;playground;school;attraction;artwork;wilderness_hut;hanami;tree_row;hedge,,>=16,+,6,16
+parking,,>=14,>=50000,6,14
```

Comment:
The [filter one level](https://github.com/tangrams/bubble-wrap/blob/gh-pages/bubble-wrap.yaml#L6492) up says `min: 50000`, so the `10000` limit is not enforced in practice.